rune/libenclave/internal/runtime/pal/api_v1_linux.go

```diff
@@ -30,13 +30,14 @@ static int palExecV1(void *sym, const char *exe, const char *argv[],
 		stdin, stdout, stderr,
 	};
 
-	return ((int (*)(const char *, const char *[], pal_stdio_fds *, int *))sym)
+	return ((int (*)(const char *, const char *[], pal_stdio_fds *, int *))
+		sym)
```

Comment:
Use indent-off.

Reply:
Can NOT add /* INDENT OFF */ in cgo code, which will result in a compile error.
But I change it to one line.